XamlStyler3.Package/StylerPackage.cs

```diff
@@ -40,20 +42,21 @@ public StylerPackage()
         {
             Trace.WriteLine(string.Format(CultureInfo.CurrentCulture, "Entering constructor for: {0}", ToString()));
         }
-        
-        protected override void Initialize()
+
+        protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
+            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
             Trace.WriteLine(string.Format(CultureInfo.CurrentCulture, "Entering Initialize() of: {0}", ToString()));
             base.Initialize();
-
-            _dte = GetService(typeof(DTE)) as DTE;
+            
+            _dte = await GetServiceAsync(typeof(DTE)) as DTE;
```

Comment:
We should address warning about checking if null for _dte and _uiShell or otherwise suppress VSSKD006 in suppression file.

Reply:
`Assumes.Present(_dte); `would be fix